docker-compose.yml

```
@@ -0,0 +1,24 @@
+version: '3.7'
+services:
+  app:
+    image: node:10-alpine
+    command: 'npm run dev'
+    working_dir: /usr/chapter/
+    volumes:
+      - data:/usr/chapter/
```

```suggestion
      - data:/usr/chapter
```

Styling - matching the path of the other service's volume

---

I've gone ahead and removed all remaining tailing slashed in PR his original pr